region/ceres/main.json

```diff
@@ -30,25 +30,33 @@
             ]
           },
           "locks": null
+        },
+        {
+          "id": 3,
+          "name": "Game Start",
+          "nodeType": "door",
+          "nodeSubType": "elevator",
+          "locks": null,
+          "note": "Game start gets its own node so that defeating Ceres Ridley is required to use the eleavtor at node 1."
```

Comment:
I don't think that's necessary. Node 1 could just be locked behind `DefeatCeresRidley`

Reply:
We never really defined what properties is locked behind locks but I guess using an elevator makes sense since other doors use it.

Reply:
I did, actually. https://github.com/miketrethewey/sm-json-data/blob/master/region/region-readme.md
See Additional Considerations in the locks section.